infrastructure/prod.tfvars

```diff
@@ -7,4 +7,3 @@ ga_tracking_id = "UA-97111056-2"
 external_host_name = "www.moneyclaims.service.gov.uk"
 feature_return_error_to_user = "false"
 capacity = "2"
-
```

Comment:
Can we use CNP fees API in prod as well?

Reply:
yup, it's over https using certs which have CA errors at the moment which should be acceptable